lang/en.json

```diff
@@ -350,129 +350,129 @@
+        "PageUrlVariableDescription": "Returns the full URL of a web page",
+        "PageUrlVariableName": "Web-page URL",
+        "PageViewTriggerDescription": "Triggered as soon as the tag manager is executed within the web page.",
         "PageViewTriggerName": "Pageview",
-        "PingdomRUMTagDescription": "Pingdom Real User Monitoring (RUM) lets you collect performance data from actual visitors to your site.",
-        "PingdomRUMTagHelp": "This tag allows you to add the Pingdom Real User Monitoring (RUM) to your site.",
+        "PingdomRUMTagDescription": "Pingdom real user monitoring (RUM) lets you collect performance data from actual visitors of your web site.",
+        "PingdomRUMTagHelp": "This tag allows adding the Pingdom real user monitoring (RUM) to your website.",
         "PluginDescription": "Manage and unify all your tracking and marketing snippets in one place.",
         "PreConfiguredInfoTitle": "Pre-configured variables are variables that are ready-to-use and cannot be created, edited, or configured. They work out of the box and you can use them in tags, triggers, and variables if needed.",
         "PreconfiguredVariables": "Pre-configured variables",
-        "PreviewDebugEnabledNotificationLine1": "Preview mode is enabled%1$s. Access your website or enter a URL to debug the container now. ",
-        "PreviewDebugEnabledNotificationLine2": "If you want to %1$sshare the preview%2$s with someone else, please append %3$s or %4$s to the URL of your website.%5$sWhile this mode is enabled, the preview container will be automatically updated when you make a change.",
-        "PreviewDebugEnabledNotificationLine3": "Debug not working? Please check this %1$sFAQ%2$s.",
+        "PreviewDebugEnabledNotificationLine1": "Preview mode is on%1$s. Access your website or enter a URL to debug the container now. ",
+        "PreviewDebugEnabledNotificationLine2": "If you want to %1$sshare the preview%2$s with someone else, please append %3$s or %4$s to the URL of your website. %5$sWhile this mode is on, the preview container will be auto-updated when you make a change.",
+        "PreviewDebugEnabledNotificationLine3": "Debugging not working? Please check this %1$sFAQ%2$s.",
         "PreviewDebugUrlButtonText": "Debug",
-        "PreviewModeVariableDescription": "Returns either \"1\" or \"0\" depending on whether the preview mode is currently enabled.",
-        "PreviewModeVariableName": "Preview Mode",
+        "PreviewModeVariableDescription": "Returns either \"1\" or \"0\" depending on whether the preview mode is on.",
+        "PreviewModeVariableName": "Preview mode",
         "Priority": "Priority",
-        "PriorityHelp": "The lower the priority is, the earlier this tag may be fired when several tags would be otherwise triggered at the same time.",
+        "PriorityHelp": "The lower the priority, the earlier this tag may be fired when several tags would otherwise be triggered at once.",
```

Comment:
```suggestion
        "PriorityHelp": "The lower the priority, the earlier this tag is fired when several tags would otherwise be triggered at once.",
```
?